ExampleApplication/ExampleApplication.csproj

```diff
@@ -1,19 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net60</TargetFrameworks>

```

Comment:
@MeenaAlfons this allowed me to test on my own mac. 

like this

`dotnet run  --framework net60`